webpack.config.js

```diff
@@ -20,11 +21,12 @@ module.exports = {
   module: {
     loaders: [
       { test: /\.ts$/, loader: 'ts-loader' },
-    ],
-    noParse: [ /angular2\/bundles\/.+/ ]
+    ]
   },
 
   devServer: {
     historyApiFallback: true
-  }
+  },
+
+  node: { crypto: 'empty' } // we need this for 'reflect-metadata' polyfill
```

Comment:
I refactored `reflect-metadata` to support universal and bundling but I ended up rebuilding half of it in the process (I even found some issues with the Map/Set polyfills) so I published it as 
[`es7-reflect-metadata`](https://github.com/gdi2290/es7-reflect-metadata)
`import 'es7-reflect-metadata/browser';`
It works great without the need for `node: { crypto: 'empty' }` and I even have better server support which I will need soon